package.json

```diff
@@ -11,29 +11,38 @@
   },
   "dependencies": {
     "async": ">=0.2",
-    "bcrypt": ">= 0.5",
+    "bcrypt": ">= 0.8.5",
     "bluebird": "^2.3.2",
-    "connect-mongo": "~0.6.0",
+    "body-parser": "~1.14.1",
+    "compression": "^1.6.1",
+    "connect-busboy": "0.0.2",
+    "connect-mongo": "~1.1.0",
+    "connect-multiparty": "^2.0.0",
     "connect-s3": "~0.3.0",
+    "cookie-parser": "1.4.x",
     "cuid": "^1.2.4",
     "ejs": "~0.8.3",
-    "express": "3.0.x",
+    "express": "4.x",
+    "express-session": "^1.11.3",
     "knox": "~0.9.1",
-    "lodash": "~1.0.1",
+    "lodash": "~3.10",
```

Comment:
Lodash v4 seems to change the behavior of `map`, which is nuts. The update isn't critical so I'm just bumping to 3 for now.